src/frontend/src/eth/components/wallet-connect/WalletConnect.svelte

```diff
@@ -2,9 +2,9 @@
 	import WalletConnectSend from './WalletConnectSend.svelte';
 	import WalletConnectSession from './WalletConnectSession.svelte';
 	import WalletConnectSign from './WalletConnectSign.svelte';
-	import type { WalletConnectListener } from '$eth/types/wallet-connect';
+	import type { OptionalNullableWalletConnectListener } from '$eth/types/wallet-connect';
```

Comment:
For consistency reasons, the prefix shouldn't be `Option` instead of `Optional` given that we already have `OptionSomething` types?

Reply:
Yes, right, changed!